lib/galaxy/managers/export_tracker.py

```diff
@@ -44,7 +44,7 @@ def set_export_association_metadata(self, export_association_id: int, export_met
             export_association: StoreExportAssociation = self.session.execute(stmt).scalars().one()
         except NoResultFound:
             raise ObjectNotFound("Cannot set export metadata. Reason: Export association not found")
-        export_association.export_metadata = export_metadata.json()
+        export_association.export_metadata = export_metadata.json()  # type:ignore[assignment]
         with transaction(self.session):
```

Comment:
```suggestion
        export_association.export_metadata = export_metadata.model_dump_json()  # type:ignore[assignment]
```

Reply:
This suggestion is completely off... I don't know what happened...

Reply:
Yeah, I see the same suggestion - `json()` is apparently deprecated. I'll try this (as a separate commit - otherwise this will remove the previous line)